src/test/java/pl/company/foodatu/FoodatuApplicationTests.java

```diff
@@ -365,6 +373,11 @@ void addMealToPlanForToday(String mealName, Map<UUID, Double> products) {
                         .map(product -> new ProductCreateDTO(product.getKey(), product.getValue()))
                         .toList()
         )).getBody();
+        try {
+            TimeUnit.SECONDS.sleep(1);
```

Comment:
Yes, I get the problem here, but sleeping for a second is not optimal as well. Each test will be longer for at least 1s now, and sometimes that may not be enough. Maybe we can use [awaitility](https://github.com/awaitility/awaitility) for that?

Reply:
I've never used awaitility before, so I think I can try it :)